UnityProject/Assets/Scripts/Objects/Machines/QuantumPad.cs

```diff
@@ -32,6 +33,8 @@ public class QuantumPad : NetworkBehaviour, ICheckedInteractable<HandApply>
 
 		private RegisterTile registerTile;
 
+		public GameObject MutatedBread;
```

Comment:
```suggestion
		[SerializeField]
		private GameObject MutatedBread = default;
```
Just keeps the prefab within the scope of QuantumPad.